### ISSUE TITLE
[Request] Playback: Play next file in the folder, then close

### ISSUE THREAD

**Issue by Dendraspis** [clsid2/mpc-hc#465]
It would be awesome, if the list of **After Playback** actions could be extended by 
***Play next file in the folder, then close***.

![MPCHC](https://user-images.githubusercontent.com/50659978/84773422-ad193e00-afdc-11ea-9f9f-5e0fc2aed7cd.JPG)

With these settings, MPC-HC plays file after file. When it reaches the last one, the screen becomes black - as long as you close MPC-HC by hand or so. Also at this mode neither Windows nor a used HomeTheatre app is able to engage the screensaver because MPC-HC *blocks*. Additionally it's sometimes weird watching on a black screen.
Long story short: It would be great having an option that lets MPC-HC close after the last file in the folder was played.

**Comment by cw2** [comment]
Could you please provide more details on how exactly are you playing multiple files? Because currently it is possible to achieve what you've described by selecting "Exit" in **After Playback** and having multiple files in the playlist (either added manually or by "Open Directory..." menu command or via command line).

I was thinking about implementing "Play next file in the folder, then..." followed by (potentially second) after playback action combobox, but that does not really make much sense, resp. such option is in my opinion more related to Playback than action after playback.

**Comment by clsid2** [comment]
Instead of adding another option we could also make it close the current file in when "play next file in folder" is selected, but there isn't a next file. That should solve the problem of screensaver being blocked. There is also no reason to keep last file opened when that after playback event is used.

**Comment by clsid2** [comment]
I have implement the automatic file close.

**Comment by ags1234** [comment]
> I have implement the automatic file close.

Implement the file close like in MPC-BE. It is very elegantly done.

![2022-04-23 19 30 19](https://user-images.githubusercontent.com/21011513/164914956-8d08d674-3697-4487-877c-1eb3e9ff9875.png)
